masschallenge/impact-api
# AC-4774: Add request tracking to impact api

## [AC-4774](https://masschallenge.atlassian.net/browse/AC-4774)

**Testing instructions**

* Start local instance: `make build && make dev`
* Make sure you have a user, then log in. (See [quickstart](https://github.com/masschallenge/impact-api/blob/development/QUICK_START.md) steps 6 and 7)
* Pull up the [API request log admin page](http://localhost:8000/admin/rest_framework_tracking/apirequestlog/).
* Using your browser, hit some DRF [low level](http://localhost:8000/api/impact/) calls
* Add [this Postman collection](https://www.getpostman.com/collections/b9dcecd3190fcd10888d) to your local Postman setup (via Collection > Import > Imporrt From Link)
* Using Postman, hit some v0 calls (e.g. "startup list")
* Using Postman, hit some v1 calls (e.g. "user list get")
* Confirm that all of the the above accesses show up in the log and have the expected information.

**NOTE:** The proxy views are not included in this work because of a crashing bug discovered in them. A new ticket(s) has been created for fixing this, under the "if this gets hairy" clause of this ticket's "DONE".
